content/api/entity-reference/fundraising/index.html

```diff
@@ -37,20 +41,152 @@
         </thead>
         <tbody>
             <tr>
-                <td>count</td>
-                <td>integer</td>
-                <td>The total number of items in the collection.</td>
+                <td>id</td>
+                <td>string</td>
+                <td>The appeal identifier.</td>
+            </tr>
+            <tr>
+                <td>description</td>
+                <td>string</td>
+                <td>The name of the appeal.</td>
+            </tr>
+            <tr>
+                <td>lookup_id</td>
+                <td>string</td>
+                <td>The user defined identifier for the appeal.</td>
+            </tr>
+            <tr>
+                <td>start_date</td>
+                <td>dateTime</td>
+                <td>The start date of the appeal. Uses <a href="https://en.wikipedia.org/wiki/ISO_8601">ISO-8601 format: </a><i>1969-11-21T10:29:43-04:00</i>.</td>
+            </tr>
+            <tr>
+                <td>end_date</td>
+                <td>dateTime</td>
+                <td>The end date of the appeal. Uses <a href="https://en.wikipedia.org/wiki/ISO_8601">ISO-8601 format: </a><i>1969-12-25T10:29:43-04:00</i>.</td>
+            </tr>
+            <tr>
+                <td>inactive</td>
+                <td>boolean</td>
+                <td>Indicates whether the appeal is active.</td>
+            </tr>
+            <tr>
+                <td>goal</td>
+                <td><a href="/constituent-entity-reference#Currency">currency</a></td>
+                <td>The monetary goal associated with the appeal.</td>
+            </tr>
+        </tbody>
+    </table>
+</div>
+
+
+<a id="Campaign"></a>
+<p>&nbsp;</p>
+<h2>Campaign</h2>
+<p>Campaigns are your overall objectives for raising money. For example, a school that's planning to build a new library could have a New Library Campaign.</p>
```

Comment:
I think we went over this line before, but if you come up with something else let me know.
Maybe something like "... to build a new library could have a campaign to raise money for the construction cost."